assets/javascripts/discourse-assign/controllers/group-assignments.js.es6

```diff
@@ -1,31 +1,28 @@
 import { inject as service } from "@ember/service";
 import Controller, { inject as controller } from "@ember/controller";
+import { ajax } from "discourse/lib/ajax";
 
 export default Controller.extend({
   router: service(),
   application: controller(),
   loading: false,
+  offset: 0,
 
-  findMembers(refresh) {
+  findMembers() {
     if (this.loading || !this.model) {
       return;
     }
 
-    if (!refresh && this.model.members.length >= this.model.user_count) {
-      this.set("application.showFooter", true);
-      return;
+    if (this.model.members.length >= this.offset + 50) {
+      this.set("loading", true);
+      this.set("offset", this.offset + 50);
+      ajax(`/assign/members/${this.groupName}?offset=${this.offset}`).then(
+        result => {
+          this.set("model.members", this.model.members.concat(result.members));
+          this.set("loading", false);
+        }
+      );
```

Comment:
Sorry I didn't mention this earlier 😓 

I don't think we should modify the `model.members` parameter. The items we are adding are not complete 'user' models, they are just simple JS objects. Can we make a `members` property in the controller, and use that instead? Make sure to reset it to an empty array each time in the `setupController` method.

Reply:
done in new commit